modules/tensor_mechanics/src/materials/NonLocalDamage.C

```diff
@@ -0,0 +1,90 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/*                       BlackBear                              */
```

Comment:
wrong header

Reply:
Nice call, I will have to change a few of these.